src/ConformanceTests.jl

```diff
@@ -11,6 +12,11 @@ function equality(a, b)
   end
 end
 
+function equality(a::Tuple, b::Tuple)
+  return all(equality.(a, b))
```

Comment:
Does broadcasting complain here when the tuples have different lengths? If not, it would be good to catch that

Reply:
Yes, it does